scripts/main.js

```diff
@@ -0,0 +1,50 @@
+var human_counter = 0;
+var machine_counter = 0;
+var letTheGameBegin_counter = 0;
+
+// debugger
```

Comment:
Avoid leaving in commented-out debuggers, console.logs etc. in your finished code